afdko/Tools/SharedData/FDKScripts/pdfutils.py

```diff
@@ -61,10 +65,10 @@ def preProcessImages(spec):
 
```

Comment:
please change line 61 from:
```python
    if type(spec) is types.StringType:
```
to:
```python
    if isinstance(spec, types.StringType):
```
(also, need to import `glob` for line 62 above)